bundles/org.openhab.binding.juicenet/src/main/java/org/openhab/binding/juicenet/internal/api/dto/JuiceNetApiDeviceChargingStatus.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.juicenet.internal.api.dto;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * {@link JuiceNetDeviceChargingStatus } implements DTO for device charging status
+ *
+ * @author Jeff James - Initial contribution
+ */
+@NonNullByDefault
+public class JuiceNetApiDeviceChargingStatus {
+    @SerializedName("amps_limit")
+    public int currentLimit;
```

Comment:
I know this is exhausting, but I would suggest to keep the API name here in the DTO and changing it only to comply with naming convention:
```suggestion
    public int ampsLimit;
```

Then DTO would be more or less 1:1 with API, and the abstraction would be in the handler which would translate from API names to channel names etc.

This applies to all DTO's.

I won't insist on this, but IMHO it would be cleaner to abstract things in the handler rather than directly in the DTO. Furthermore, you can't assume that each DTO member translates directly to a channel anyway. In some cases you might need some conversion or mapping. As an example, it could be relevant to convert miles to kilometers.